openc3/lib/openc3/utilities/store_queued.rb

```diff
@@ -0,0 +1,113 @@
+# encoding: ascii-8bit
+
+# Copyright 2024 OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+
+require 'openc3/utilities/store'
+
+module OpenC3
+  class StoreQueued
+    # How often in seconds the store is updated
+    UPDATE_INTERVAL = 2
```

Comment:
why 2?

Reply:
This needs to be an argument to the instance on first use (like System).   I want to use this same class to implement #987